packages/editor/src/validator/rules/PropertiesRules.ts

```diff
@@ -14,21 +16,17 @@ class PropertySchemaValidatorRule implements PropertiesValidatorRule {
     component,
     trait,
     validators,
+    stateManager
```

Comment:
The validator is for static validation by design, so we can not add `stateManager` in validator.

Reply:
Should we display the runtime expression errors in the waring area?

Reply:
Maybe not now. We'd better focus on expression editor this time.

Reply:
OK, I remove the codes.